lib/ts/recipe/openid/api/implementation.ts

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import { APIInterface, APIOptions } from "../types";
+
+export default function getAPIImplementation(): APIInterface {
+    return {
+        getOpenIdDiscoveryConfigurationGET: async function ({
+            options,
+        }: {
+            options: APIOptions;
+        }): Promise<{ status: "OK"; issuer: string; jwks_uri: string }> {
```

Comment:
You have types declared somewhere, but you are not using it here.

Reply:
As discussed, removed the created type and using the object representation type everywhere